fdbclient/tests/bulkload_test.sh

```diff
@@ -19,6 +20,18 @@ trap cleanup  EXIT
 
 # Cleanup. Called from signal trap.
 function cleanup {
+  # Check if test data should be preserved (from tests_common.sh)
+  if should_preserve_test_data; then
+    # Shutdown servers but don't delete data
+    if type shutdown_fdb_cluster &> /dev/null; then
+      shutdown_fdb_cluster
+    fi
+    if type shutdown_mocks3 &> /dev/null; then
+      shutdown_mocks3
+    fi
+    return 0
+  fi
+  
```

Comment:
I see the same code in `s3_backup_test.sh`. Can you refactor them into a common file?